scripts/create_test_subset.py

```diff
@@ -627,6 +658,30 @@ def get_sids_for_families(
     return included_sids
 
 
+def get_sids_for_cohorts(
+    project: str, cohorts: set[str], cohort_samples_n: int
+) -> set[str]:
+    """Returns cohort_samples_n specific samples for given cohort IDs."""
+
+    cohort_sid_output = query(COHORT_QUERY, {'project': project})
+
+    all_cohort_groups = cohort_sid_output.get('project', {}).get('cohorts', [])
+
+    all_cohorts_sample_ids_subset: set[str] = set()
+    for cohort in all_cohort_groups:
+        sids_for_cohort: list[str] = []
```

Comment:
I guess this list construction can go after the check on line 673.
` if cohort.get('id') in cohorts:`

There is no need to construct empty list when looping through cohort which is not on the cohorts list.